apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -105,23 +107,17 @@ export function InstructorQuestionSettings({
                     id="title"
                     name="title"
                     value="${resLocals.question.title}"
-                    disabled
+                    ${resLocals.authz_data.has_course_permission_edit &&
+                    !resLocals.course.example_course
+                      ? ''
+                      : 'disabled'}
```

Comment:
Consider adding the same `canEdit` boolean from the assessment settings PR.

Reply:
Added `canEdit` in [4fd6719](https://github.com/PrairieLearn/PrairieLearn/pull/10698/commits/4fd671935566a70b740a03f5534e17952908fb58).